File: samples/CS/01-AddingRibbonSupport/Form1.cs
using System;
using System.Windows.Forms;
using WinForms.Ribbon;

namespace _01_AddingRibbonSupport
{
    public partial class Form1 : Form
    {
        private RibbonItems _ribbonItems;

        public Form1()
        {
            InitializeComponent();
            _ribbonItems = new RibbonItems(_ribbon);
            _ribbonItems.Init();
        }
    }
}

File: samples/VB/14-ContextualTabs/Form1.vb
Imports WinForms.Ribbon

Namespace _14_ContextualTabs

    Partial Public Class Form1
        Inherits Form
        Private _ribbonItems As RibbonItems

        Public Sub New()
            InitializeComponent()
            _ribbonItems = New RibbonItems(_ribbon)
            _ribbonItems.Init()

        End Sub
    End Class
End Namespace
